user_tools/docs/user-tools-onprem.md

```diff
@@ -0,0 +1,130 @@
+# RAPIDS User Tools on OnPrem platform
+
+This is a guide for the RAPIDS tools for Apache Spark on OnPrem platform. At the end of this guide, the user will be able to run the RAPIDS tools to analyze the clusters and the applications running OnPrem.
```

Comment:
I am not sure it is accurate to say `OnPrem platform`.
Maybe we should rephrase t o elaborate what we mean by OnPrem just in case someone can confuse it that it is some sort of a cloud product. for example
```suggestion
This is a guide for the RAPIDS tools for Apache Spark clusters that are provisioned manually (a.k.a onPrem). At the end of this guide, the user will be able to run the RAPIDS tools to analyze the clusters and the applications running OnPrem.
```